PlayerTest.py

```diff
@@ -0,0 +1,54 @@
+import player
+from tables import INFINITY
+import unittest
+
+
+class PlayerTest(unittest.TestCase):
+    def test_add_marbles(self):
+        player1 = player.Player('Glenn')
+        self.assertEqual(player1.marbles, {'b': 0, 'g': 0, 'w': 0})
```

Comment:
Need to check if Python allows an assertEqual between two dicts.  Actually reading it appears that it calls assertDictEqual and so you might be okay.